package.json

```diff
@@ -4,12 +4,17 @@
   "description": "VCs for Elia Exchange of EnergyBlocks",
   "main": "index.js",
   "scripts": {
+    "check-engine": "node -e \"if (!require('semver').satisfies(process.version, require('./package.json').engines.node)) { console.error('Unsupported Node.js version! Please use a version that matches: ' + require('./package.json').engines.node); process.exit(1) }\"",
+    "pretest": "npm run check-engine",
```

Comment:
I think that this is not necessary as npm supports node engine version checking when having a project set as described in [my comment](https://github.com/energywebfoundation/elia-energyblocks-vcs/pull/23#issuecomment-1737752278)

Reply:
I added the `.npmrc` file with `engine-strict=true` in this commit: https://github.com/energywebfoundation/elia-energyblocks-vcs/pull/23/commits/ef34863b3f98c8aa88d9296aff670f20c2ca7d09
However, I think this is only checked when running `npm install`, not when running npm scripts. I was still able to run `npm test` when using node v18.

Reply:
ah, you are right. I don't know why I had the impression it is verified for all npm scripts. So nevermind :)